tensorflow/lite/micro/kernels/reduce_test.cc

```diff
@@ -128,6 +128,48 @@ void TestMeanFloatInput4D(const int* input_dims_data, const float* input_data,
                             output_data, output_dims_count, params, tolerance));
 }
 
+template <typename T>
+void TestMeanOpQuantized(const int* input_dims_data, const float* input_data,
+                         float input_scale, int input_zero_point,
+                         const int* axis_dims_data, const int32_t* axis_data,
+                         const int* output_dims_data,
+                         const float* expected_output_data, float output_scale,
+                         int output_zero_point, TfLiteReducerParams* params) {
+  // Convert dimesion arguments to TfLiteArrays
+  TfLiteIntArray* input_dims = IntArrayFromInts(input_dims_data);
+  TfLiteIntArray* axis_dims = IntArrayFromInts(axis_dims_data);
+  TfLiteIntArray* output_dims = IntArrayFromInts(output_dims_data);
+
+  // Get number of elements in input and output tensors
+  const int output_dims_count = ElementCount(*output_dims);
+  const int input_dims_count = ElementCount(*input_dims);
+
+  // Allocate arrays for quantized tensors
+  T output_data_quant[output_dims_count];
```

Comment:
This is blocked in Google internal toolchain.
error: variable length array used [-Werror,-Wvla]

Please fix other lines too.

Reply:
fixed